server/models/Request.model.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user: { type: mongoose.ObjectId, required: true, ref: "user" },
```

Comment:
`owner` might be a more descriptive alternative as both `sitter/owner` are users.